__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const db = require("../db/connection.js");
+const { app } = require("../app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+const { response } = require("express");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(testData); // <- seeding each time DELETE COMMENT later!
+});
+
+describe("Using app.js to run the database of NC-games", () => {
```

Comment:
-dersThink about the structure of describe/test blocks.  Is this first describe helping us understand what the tests are doing?

There is no prescribed way of doing it, but an example could be:
describe("GET api/categories")
test("200: returns array of category objects")

Reply:
Okay that's understandable